lib/ngx/pipe.md

```diff
@@ -314,9 +314,8 @@ a string describing the error.
 If the `merge_stderr` option is specified in [spawn](#spawn), closing the
 `stderr` direction will return `nil` and the error string `"merged to stdout"`.
 
-Shutting down a direction when a light thread is waiting on it (such as during
-reading or writing) will return `nil` and the error string `"pipe busy
-writing"` (for stdin) or `"pipe busy reading"` (for the others).
+Shutting down a direction when there is a light thread waiting on it (like
+reading or writing) will abort the light thread and return `true`.
```

Comment:
This English sentence has grammatical errors.

Reply:
@agentzh 
Fixed in the new commit.